### PR TITLE
Allow non-local Label Studio instances for annotation

### PR DESCRIPTION
**Pull request by strickvl**
I updated the label studio annotator stack component to allow for deployed (i.e. non-local) instances of Label Studio to be used in stacks / ZenML.

## Pre-requisites
Please ensure you have done the following:
- [x] I have read the **CONTRIBUTING.md** document.
- [ ] If my change requires a change to docs, I have updated the documentation accordingly.
- [ ] If I have added an integration, I have updated the [integrations](https://docs.zenml.io/component-gallery/integrations) table and the [corresponding website section](https://zenml.io/integrations).
- [ ] I have added tests to cover my changes.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Other (add details above)

### CODE REVIEW

**Review by fa9r — APPROVED** [review]

**Review by htahir1 — COMMENTED** [review]
Cool, easy win. Just one comment

File: src/zenml/integrations/label_studio/flavors/label_studio_annotator_flavor.py [hunk 1]
```diff
@@ -26,7 +26,8 @@
     from zenml.integrations.label_studio.annotators import LabelStudioAnnotator
 
 
-DEFAULT_LABEL_STUDIO_PORT = 8093
+DEFAULT_LOCAL_INSTANCE_URL = "https://localhost"
```
[the remark this answers is not in the document]
Comment (strickvl) [reply src/zenml/integrations/label_studio/flavors/label_studio_annotator_flavor.py:29]: You're totally right. Good catch!

File: src/zenml/integrations/label_studio/annotators/label_studio_annotator.py [hunk 2]
```diff
@@ -87,7 +88,10 @@ def get_url(self) -> str:
         Returns:
             The URL of the annotation interface.
         """
-        return f"http://localhost:{self.config.port}"
+        if self.config.instance_url == DEFAULT_LOCAL_INSTANCE_URL:
```
Comment (htahir1) [inline src/zenml/integrations/label_studio/annotators/label_studio_annotator.py:91]: Why do we need this condition? Isn't it okay just to return self.config.instance_url?

Comment (strickvl) [reply src/zenml/integrations/label_studio/annotators/label_studio_annotator.py]: Not quite, but you were right that there was a simpler way to do this. I encountered some errors when testing it out on a deployed instance earlier, but I realised what I was doing wrong. I removed the conditional now and updated the docs to make it clearer what the user needs to do.

**Review by schustmi — APPROVED** [review]

**Pull request merged**